### PR TITLE
Feature 2

### PR DESCRIPTION
**Pull request by asantos2000**
Revised changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1585,7 +1585,328 @@ git commit -m "Change README pull oper"
 #  1 file changed, 6 insertions(+)
 
 # Bring remote changes
-git pull -vv
```
Comment (adsantos2000) [inline README.md:1588]: Still too long

File: README.md [hunk 2]
```diff
@@ -1585,7 +1585,328 @@ git commit -m "Change README pull oper"
...
+# ...
+
+# Try to push new branch
+git push -v
+# fatal: The current branch feature-2 has no upstream branch.
+# To push the current branch and set the remote as upstream, use
+
+#     git push --set-upstream origin feature-2
+
+# git push -u origin feature-2 # The same of
```
Comment (adsantos2000) [inline README.md:1688]: Ok explain two options

**Review by adsantos2000 — CHANGES_REQUESTED** [review]
Check first section

File: README.md (hunk 1 above)
Comment (asantos2000) [reply README.md:1588]: Ok, next revision I will split it

**Review by adsantos2000 — APPROVED** [review]
It's all ok for the draft version

**Comment by asantos2000** [comment]
First revision, still draft

**Pull request merged**